server/tserver/src/main/java/org/apache/accumulo/tserver/session/MultiScanSession.java

```diff
@@ -43,11 +45,12 @@
 
   public MultiScanSession(TCredentials credentials, KeyExtent threadPoolExtent,
       Map<KeyExtent,List<Range>> queries, ScanParameters scanParams,
-
-      Map<String,String> executionHints) {
-    super(credentials, scanParams, executionHints);
+      Map<String,String> executionHints, TabletResolver tabletResolver) {
+    super(credentials, scanParams, executionHints, tabletResolver);
     this.queries = queries;
     this.threadPoolExtent = threadPoolExtent;
+    // TODO this is only needed for scan server
```

Comment:
I don't think there is anything to do here. We could clean up ScanSession and MultiScanSession, but that could also be done in another PR

Reply:
oh was thinking of only making the set copy if its scan server.. I will look at this tomorrow.